src/Uno.UI/Extensions/VectorExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Text;
+using Windows.Foundation;
+
+namespace Uno.UI.Extensions
+{
+	public static class VectorExtensions
```

Comment:
Are those extensions on a `Windows.*` namespace in WinUI ?

Reply:
In UWP these extensions come from the [nuget package](https://www.nuget.org/packages/System.Numerics.Vectors.WindowsRuntime/) `System.Numerics.Vectors.WindowsRuntime`

Reply:
> Are those extensions on a `Windows.*` namespace in WinUI ?

In UWP these extensions come from the [nuget package](https://www.nuget.org/packages/System.Numerics.Vectors.WindowsRuntime/) `System.Numerics.Vectors.WindowsRuntime`

Reply:
We probably should be using the same namespace for the extension class.

Reply:
@jeromelaban  updated the namespace to match the one in the original package